pkg/controllermanager/controller/controllerregistration/seed/reconciler.go

```diff
@@ -488,6 +489,11 @@ func deployNeededInstallation(
 			deploymentSpecHash := utils.HashForMap(deploymentMap)[:16]
 			metav1.SetMetaDataLabel(&controllerInstallation.ObjectMeta, ControllerDeploymentHash, deploymentSpecHash)
 		}
+
+		if podSecurityEnforce, ok := controllerRegistration.Annotations[v1beta1constants.AnnotationPodSecurityEnforce]; ok {
+			metav1.SetMetaDataAnnotation(&controllerInstallation.ObjectMeta, v1beta1constants.AnnotationPodSecurityEnforce, podSecurityEnforce)
+		}
```

Comment:
We probably should also remove it again if necessary, or?
```suggestion
		if podSecurityEnforce, ok := controllerRegistration.Annotations[v1beta1constants.AnnotationPodSecurityEnforce]; ok {
			metav1.SetMetaDataAnnotation(&controllerInstallation.ObjectMeta, v1beta1constants.AnnotationPodSecurityEnforce, podSecurityEnforce)
		} else {
			delete(controllerInstallation.Annotations, v1beta1constants.AnnotationPodSecurityEnforce)
		}
```